public_html/wp-content/mu-plugins/blocks/assets/src/sponsors/block-content.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies.
+ */
+import { get, difference } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies.
+ */
+const { Component } = wp.element;
+
+/**
+ * Internal dependencies.
+ */
+import FeaturedImage from '../shared/featured-image';
+import GridContentLayout from '../shared/grid-layout/block-content';
+import { ItemTitle, ItemHTMLContent } from '../shared/block-content';
+
+/**
+ * Renders individual sponsor post inside editor.
+ *
+ * @param sponsorPost
+ * @param attributes
+ * @param onFeatureImageChange
+ * @returns {*}
+ * @constructor
+ */
+function SponsorDetail( { sponsorPost, attributes, onFeatureImageChange } ) {
+
+	const {
+		show_name, show_logo, show_desc
+	} = attributes;
+
+	const featuredImageSizes = get( sponsorPost, "_embedded.wp:featuredmedia[0].media_details.sizes", {} );
+
+	return (
+		<div className={ "wordcamp-sponsor-details"}>
+
+			{ ( show_name || show_name === undefined ) &&
+			<ItemTitle
+				className='wordcamp-sponsor-title'
+				headingLevel={ 3 }
+				title={ sponsorPost.title.rendered.trim() }
+				link={ sponsorPost.link }
+			/>
+			}
+			{ ( show_logo || show_logo === undefined ) &&
+			<FeaturedImage
+				className={"wordcamp-sponsor-featured-image wordcamp-sponsor-logo"}
+				wpMediaDetails={featuredImageSizes}
+				alt={sponsorPost.title.rendered}
+				attributes={attributes}
+			/>
+			}
+			{ ( show_desc || show_desc === undefined ) &&
+			<ItemHTMLContent
+				className={ classnames( 'wordcamp-sponsor-content' ) }
+				content={ sponsorPost.content.rendered.trim() }
+			/>
```

Comment:
`[SHOULD]` It looks like the design calls for excerpts support, similar to Speakers. Did you leave that out because the post type doesn't support it? It seems like we should enable support in the post type instead.

Reply:
Opened new issue for this https://github.com/WordPress/wordcamp.org/issues/38